# Motors
num_motors: 2

motors:
  motor_0:
    id: 2
    model: "XM430_W350"
  motor_1:
    id: 3
    model: "XM430_W350"
